sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/implementation/util/Transforms.java

```diff
@@ -84,6 +84,7 @@ public class Transforms {
     public static AnalyzeResult toAnalyzeResultOperation(
         com.azure.ai.formrecognizer.documentanalysis.implementation.models.AnalyzeResult innerAnalyzeResult) {
         AnalyzeResult analyzeResult = new AnalyzeResult();
+        AnalyzeResultHelper.setServiceVersion(analyzeResult, innerAnalyzeResult.getApiVersion());
```

Comment:
Just double checking but these `getApiVersion` calls are internal, and the exposed methods are called `getServiceVersion`, is that right?